mlflow/server/js/.storybook/main.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Do we need files in `.storybook`?

Reply:
Probably yes, [storybook](https://storybook.js.org/) allows people to start add stories to components -- we probably can benefit from doing that in OSS too.

Reply:
makes sense :)